wallet/src/de/schildbach/wallet/ui/CreateUsernameActivity.kt

```diff
@@ -289,7 +289,9 @@ class CreateUsernameActivity : InteractionAwareActivity(), TextWatcher {
                     }
                 })
                 showProcessingState()
-                startService(CreateIdentityService.createIntentFromInvite(this, username, invite))
+                invite?.let {
```

Comment:
Kotlin 1.5 requires stricter null-safety, `invite` doesn't fit into `createIntentFromInvite` since it's nullable.